plugins/UM3NetworkPrinting/src/Network/LocalClusterOutputDeviceManager.py

```diff
@@ -232,6 +232,9 @@ def _createMachineFromDiscoveredDevice(self, device_id: str) -> None:
         self._connectToOutputDevice(device, new_machine)
         self._showCloudFlowMessage(device)
 
+        _abstract_machine = CuraStackBuilder.createAbstractMachine(device.printerType)
```

Comment:
should this have been implemented in the previous ticket? anyway nice catch!

Reply:
This is for the LAN printers which wasn't in the requirements for the last ticket (by accident I assume).